src/rpc/rawtransaction.cpp

```diff
@@ -924,10 +924,19 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
 
     TxValidationState state;
     bool test_accept_res;
+    CAmount fee{0}; // To return transaction fee from AcceptToMemoryPool
     {
         LOCK(cs_main);
         test_accept_res = AcceptToMemoryPool(mempool, state, std::move(tx),
-            nullptr /* plTxnReplaced */, false /* bypass_limits */, max_raw_tx_fee, /* test_accept */ true);
+            nullptr /* plTxnReplaced */, false /* bypass_limits */, /* test_accept */ true, &fee);
```

Comment:
A follow-up could be to modify `testmempoolaccept` signature to return the computed fee/feerate, do we already have a RPC to evaluate-fee-but-don't-submit-to-mempool  ?

Reply:
`testmempoolaccept` would be the best place for this imo, and it is being implemented in #19093 btw!